src/utils/components/Toggles/DropdownToggle.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Ref } from 'react';
+
+import { MenuToggle, MenuToggleElement, MenuToggleProps } from '@patternfly/react-core';
+
+const DropdownToggle = ({ children, ...props }: MenuToggleProps) =>
+  function DToggle(toggleRef: Ref<MenuToggleElement>) {
```

Comment:
why mixing both arrow function and native function syntax? Please stick with arrow function notation

Reply:
I transformed the inside arrow function into a native function as typescript error raise about react component without `displayName`